### ISSUE TITLE
Generate-Config.ps1: App Insights Instrumentation Key is output twice, unnecessary

### ISSUE THREAD

**Issue by petertuton** [microsoft/TailwindTraders-Backend#80]
Deploy/powershell/Generate-Config.ps1 unnecessary outputs the App Insights Instrumentation Key twice. Recommend deleting the first call to `Write-Host "App Insights Instrumentation Key: $($appinsightsId)" -ForegroundColor Yellow`

**Comment by petertuton** [comment]
E.g: 
```
===========================================================
gvalues file will be generated with values:
App Insights Instrumentation Key: xxx
App Insights Instrumentation Key: xxx
...
```

**Comment by olgamarti** [comment]
Thanks for try our demo. This issue is fixed in #84 . Please, if you are still experiencing this problem, re-open the issue.
